db/migrate/20210222102503_create_users_settings.rb

```diff
@@ -20,7 +20,6 @@ def change
 
       t.boolean "permit_adjacent_sponsors", default: true
       t.string "brand_color1", default: "#000000"
-      t.string "brand_color2", default: "#ffffff"
```

Comment:
This should not be removed. Migration files are records of changes so best not to touch them.

Reply:
This makes sense. This change has been reverted.